index.js

```diff
@@ -6,18 +6,23 @@ module.exports = function (opts) {
   require('./lib/array_qsort');
   require('./lib/array_unique');
   require('./lib/fs_scandir');
-  require('./lib/object_watch');
+
   require('./lib/string_format');
   require('./lib/string_hashcode');
 
   // skip prototype pollution of methods used by browserify
   if (!opts.build) {
     require('./lib/string_pipe');
   }
+  if (!opts.test) {
+    require('./lib/object_watch');
+  }
 
   return {
-    flatten   : require('./lib/flatten'),
-    get       : require('./lib/get_deep'),
-    set       : require('./lib/set_deep')
-  };
+      flatten   : require('./lib/flatten'),
+      get       : require('./lib/get_deep'),
+      set       : require('./lib/set_deep')
+    };
 };
+
+module.exports.fs = require('./lib/fs_scandir');
```

Comment:
This require does not return anything, it simply extends the `fs-extra` module & this is the second time it's being required in this file.